packages/boot/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -256,8 +256,8 @@ test.serial('makes usdc advance', async t => {
   harness?.resetRunPolicy();
 
   const doc = {
```

Comment:
Request: Since this is now capdata and not a plain string, please use https://github.com/Agoric/agoric-sdk/blob/33173f9535005998b7ea7f44aa6ac3b81ba09085/packages/boot/test/fast-usdc/fast-usdc.test.ts#L158

Or even the new `unserializePureData` function

Reply:
I made a `defaultSerializer.parse` and used it where the data were marshalled. `fastUsdc.feedPolicy` was not.

Reply:
Why not unserialize fastUsdc.txns also though? It appears to be capdata from the snapshot:

```
{"body":"#{\\"status\\":\\"ADVANCING\\"}","slots":[]}
```

Reply:
oops, thought I had